DNN Platform/Library/Services/Installer/Installers/ScriptInstaller.cs

```diff
@@ -271,6 +299,10 @@ protected override void ProcessFile(InstallFile file, XPathNavigator nav)
                 {
                     this._upgradeScript = file;
                 }
+                else if (file.Name.StartsWith("schema.", StringComparison.InvariantCultureIgnoreCase))
+                {
+                    this._schemaScripts.Add(file);
```

Comment:
Your description said that these would run in alphabetical order, but it looks like this would be in source order.  I think source order is the right choice, I just wanted to point out that it's different than what you described, unless I'm missing something.

Reply:
You're right. They run in the order they appear in the manifest.